setup.py

```diff
@@ -32,6 +30,6 @@
      , version = version
      , zip_safe = False
      , package_data = {'aspen': ['request_processor/mime.types']}
-     , install_requires = ASPEN_DEPS
-     , tests_require = TEST_DEPS
+     , install_requires = open('requirements.txt').read()
+     , tests_require = open('tests/requirements.txt').read()
```

Comment:
I hadn't thought of that, but it looks like they're being added to the `sdist` tarball automatically somehow. However I've found an easier solution that doesn't rely on magic, so I've forced-pushed it here. Previous HEAD was 7c4f8afdcb590f85e91d1a873b7a742156f118ac.